tests/dialects/test_arith.py

```diff
@@ -69,10 +67,8 @@
 from xdsl.utils.isattr import isattr
 from xdsl.utils.test_value import TestSSAValue
 
-_BinOpArgT = TypeVar("_BinOpArgT", bound=Attribute)
-
 
-class Test_integer_arith_construction:
+class test_integer_arith_construction:
     operand_type = i32
```

Comment:
minor renaming.

Reply:
Don't we normally go with `TestIntegerArithConstruction` for classes

Reply:
Do we? I hadn't noticed. Let me see and I'll update it.